types/zwjsbridge/tsconfig.json

```
@@ -0,0 +1,16 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": ["es6", "dom"],
```

```suggestion
        "lib": ["es6"],
```
your `index.d.ts` does not depend on `dom` lib. The tests are, as you added `console` symbol.
Please replace console with types assertions, or just do not use console:
https://github.com/DefinitelyTyped/DefinitelyTyped#my-package-teststs